db/projects/adex/project.yaml

```diff
@@ -0,0 +1,136 @@
+name: AdEx
+web:
+  - 'https://www.adex.network/'
+assets:
+  - id: adx
+    name: AdEx
+    type: token
+    ledger: ethereum
+    symbol: ADX
+    contract_address: '0x4470bb87d77b963a013db939be332f927f2b992e'
+    decimals: 4
+    max_supply: 100000000
+    start_date: '2017-05-30'
+    webids:
+      coinmarketcap: adx-net
+      cryptocompare: adx
+      cryptocompare/id: '170452'
+    ico_properties:
+      start_date: '2017-06-30'
+      total_cap: 11000000
+    first_distribution:
+      - title: 'team'
+        amount: 10000000
+        percent: 10
+      - title: 'advisor'
+        amount: 6000000
+        percent: 6
+      - title: 'promotion'
+        amount: 2000000
+        percent: 2
+      - title: 'bounty'
+        amount: 2000000
+        percent: 2
+      - title: 'public sale'
```

Comment:
define common types of distribution - same type of list as we planning for Proff_of_XYZ

Reply:
will be working on it